public/js/utils.js

```diff
@@ -223,18 +223,35 @@ function formatDate(d) {
     );
 }
 
+/**
+ * Returns formatted date 'Day Mon dd yyyy, hh:mm {am|pm}'
+ * @param  {Date} d - the date to be formatted
+ */
+function formatDate(d) {
+    //include 0 if the year is less than 10
+    const year = d.getFullYear() < 10 ? "0" + d.getFullYear() : d.getFullYear();
+    //include 0 if the month is less than 10
+    const month = d.getMonth()+1 < 10 ? `0${d.getMonth()+1}` : d.getMonth()+1;
+    //include 0 if the day is less than 10
+    const day = d.getDate() < 10 ? `0${d.getDate()}` : d.getDate();
+
+    return `${year}/${month}/${day} ${formatAMPM(d)}`;
```

Comment:
Must be `${month+1}`

Even though the fix is in PMS-4, lets apply this para maayos na pag merge sa development.

Reply:
see [ff9f289](https://github.com/erik-lance/TitafindsXG5/pull/11/commits/ff9f28905aaea1d93ce0ee0f4de2ade76e361ff1)